governance.md

```diff
@@ -0,0 +1,164 @@
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
+The purpose of a SIG is to own and develop a set of subprojects.
+
+A SIG can have its own policy for contribution, described in a README or CONTRIBUTING file in the SIG folder in this repo (e.g. sig-community/CONTRIBUTING.md), and its own mailing list, chat channel, etc.
+
+#### Governance 1.1 future groups
+[These descriptions](https://github.com/kubernetes/community#governance) are borrowed from [Kubernetes governance around community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups).
+They are good-enough placeholders for discussions.
+
+* **Committees** are named sets of people that are chartered to take on sensitive topics. This group is encouraged to be as open as possible while achieving its mission but, because of the nature of the topics discussed, private communications are allowed. Examples of committees include the Technical Steering Committee or an overarching Steering Committee, and things like security or code of conduct.
+* **Subprojects** Each SIG can have a set of subprojects. These are smaller groups that can work independently. Some subprojects will be part of the main Kubernetes deliverables while others will be more speculative and live in the kubernetes-sigs github org.
+* **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries. Working groups do not own any code or other long term artifacts. Working groups can report back and act through involved SIGs.
+* **User Groups** are groups for facilitating communication and discovery of information related to topics that have long term relevance to large groups of Kubernetes users. They do not have ownership of parts of the Kubernetes code base.
+See the full governance doc for more details on these groups.
+
+### Decision making
+
+While the project is in the early stages, we are favoring a balance of including all voices and keeping a rapid pace of progress.
+To accomplish this balance, we are using a _consensus decision model_ that gives time and space for everyone to voice concerns with the proposal.
+
+There are two types of consensus models we use, the lightweight and the full:
+
+* **Lightweight consensus** occurs when a decision does not have wide or sweeping effects and is considered approved if no one raises any blocking objections after several days.
+In practice, an email is sent to a SIG or main community list, and if no one raises an objection after three days, the proposal passes.
```

Comment:
Does someone need to note it's a proposal? Or transparently note that it has passed?

Reply:
s/3 days/3 work days/

Reply:
```suggestion
In practice, a proposal email with sufficient detail is sent to a SIG or main community list, and if no one raises an objection after three working days (i.e., Saturdays and Sundays are not counted), the proposal passes.
```